src/display/api.js

```diff
@@ -626,6 +626,21 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
     getMetadata: function PDFDocumentProxy_getMetadata() {
       return this.transport.getMetadata();
     },
+    /**
+     * @param {number} pageNumber The page number to get the page size from.
+     * The first page is 1, which is also the default page used.
+     * @return {Promise} A promise that is resolved with an array of the
+     * width and height in the user space units - [width, height].
+     */
+    getPageSize(pageNumber) {
+      pageNumber = pageNumber || 1;
+      return this.getPage(pageNumber).then((page) => {
+        return [
+          page.pageInfo.view[2] - page.pageInfo.view[0], // width
+          page.pageInfo.view[3] - page.pageInfo.view[1]  // height
```

Comment:
Don't use internal `pageInfo`, use `view` directly. I recommend hoisting it above, e.g. via `const [x1,y1,x2,y2] = page.view;`. Also, [user unit](https://github.com/mozilla/pdf.js/blob/master/src/display/api.js#L866) is not used.